### PR TITLE
Rel

### PR DESCRIPTION
**Pull request by clayrat**

### CODE REVIEW

**Comment by clayrat** [comment]
Ready for review.

File: src/Rel.lidr
Comment (yurrriq) [inline src/Rel.lidr]:
```tex
\idr{Total_relation}
```

File: src/Rel.lidr
Comment (yurrriq) [inline src/Rel.lidr]:
```tex
\idr{Empty_relation}
```

File: src/Rel.lidr
Comment (yurrriq) [inline src/Rel.lidr:314]: Note for later: minted handles this definition poorly. See also #30.

**Review by yurrriq — CHANGES_REQUESTED** [review]
We'll need to add `Rel` to `software_foundations.ipkg` and `Rel.lidr` to `src/Makefile` too.

**Comment by yurrriq** [comment]
Other than those little comments this looks good to me.

**Comment by yurrriq** [comment]
Feel free to merge this one yourself after those trivial changes.

File: src/Rel.lidr
Comment (clayrat) [reply src/Rel.lidr:314]: So what should we do here? Add a `todo`?

**Comment by yurrriq** [comment]
Sure, though linking the relevant issue in my previous  comment might be enough. This is a toolchain issue rather than a code one.

**Comment by clayrat** [comment]
Let's merge `ProofObjects` first maybe and add to `.ipkg`/`Makefile` after that? I feel like it'll be a merge conflict anyway.

**Review by yurrriq — APPROVED** [review]

**Pull request merged**
